db/migrate/20160414063050_add_sti_to_subnets.rb

```diff
@@ -0,0 +1,11 @@
+class AddStiToSubnets < ActiveRecord::Migration
+  def self.up
+    add_column :subnets, :type, :string, :default => 'Subnet::Ipv4'
```

Comment:
if you set type `:null => false` which I'd recommend anyway, I think you don't have to explicitly do the update query, it should set the default value